### PR TITLE
fix: Make `flame_3d` work with latest stable

### PR DESCRIPTION
**Pull request by wolfenrain**
# Description

Fixes up the changes needed in flame_3d to work with the latest version of Flutter stable.

## Checklist

- [ ] I have followed the [Contributor Guide] when preparing my PR.
- [ ] I have updated/added tests for ALL new/updated/fixed functionality.
- [ ] I have updated/added relevant documentation in `docs` and added dartdoc comments with `///`.
- [ ] I have updated/added relevant examples in `examples` or `docs`.

## Breaking Change?

- [ ] Yes, this PR is a breaking change.
- [ ] No, this PR is not a breaking change.

## Related Issues

[Contributor Guide]: https://github.com/flame-engine/flame/blob/main/CONTRIBUTING.md
[Conventional Commit]: https://conventionalcommits.org
[CHANGELOG]: https://github.com/flame-engine/flame/blob/main/CHANGELOG.md

### CODE REVIEW

File: .github/workflows/cicd.yml [hunk 1]
```diff
@@ -43,7 +45,7 @@ jobs:
       - name: "Analyze with latest stable"
         uses: invertase/github-action-dart-analyzer@v3
         with:
-          fatal-infos: true
+          fatal-infos: false # To prevent deprecation messages from failing.g
```
Comment (Salakar) [inline .github/workflows/cicd.yml:48]: These should be fixed now right?

Comment (wolfenrain) [inline .github/workflows/cicd.yml:48]:
```suggestion
          fatal-infos: true
```

**Review by luanpotter — APPROVED** [review]
LGTM

File: .github/workflows/cicd.yml [hunk 2]
```diff
@@ -29,6 +29,8 @@ jobs:
         with:
           flutter-version: ${{env.FLUTTER_MIN_VERSION}}
       - uses: bluefireteam/melos-action@v3
+      # flame_3d has a higher lowest supported version so we just remove it here to prevent CI errors
+      - run: rm -rf packages/flame_3d
```
Comment (wolfenrain) [inline .github/workflows/cicd.yml:33]:
```suggestion
```

**Pull request merged**